tools/topology/topology2/include/pipelines/cavs/be-dai.conf

```diff
@@ -33,8 +33,8 @@ Class.Pipeline."be-dai" {
 		]
 
 		#
-		# mixin-playback objects instantiated within the same alsaconf node must have
-		# unique pipeline_id attribute
+		# be-dai objects instantiated within the same alsaconf node must have
+		# unique instance attribute
```

Comment:
that's even more confusing for the case of an SSP, we need two instance attributes for the same SSP.